.changeset/tender-buckets-confess.md

```diff
@@ -0,0 +1,8 @@
+---
+"wrangler": patch
+---
+
+feat: expose `--show-interactive-dev-session` flag
+
+This flag disables the interactive mode of the dev session, a feature that already exists internally but was not exposed to the user.
```

Comment:
Strictly it allows you to disable interactive mode by setting `--show-interactive-dev-session=false`.
Perhaps it can just say:
```suggestion
This flag controls the interactive mode of the dev session, a feature that already exists internally but was not exposed to the user.
```

Reply:
Sure, have updated in [4d374f9](https://github.com/cloudflare/workers-sdk/pull/5099/commits/4d374f91b220510df6fe77c1900ea446879a5cb8)